elixir-sdk/test/sdk_core_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule SdkCoreTest do
+  use ExUnit.Case
+
+  test "init with valid config succeeds" do
+    config = %SdkCore.Config{
+      api_key: "test-key",
+      base_url: "https://api.eppo.cloud",
```

Comment:
> 3) How do you mock a configuration so it is possible to write more tests locally?

We have a [mock-server](https://github.com/Eppo-exp/eppo-multiplatform/blob/21526f774f7cd03fdb2d2a29e4a6321a76967eb0/mock-server/) implementation that exposes all configurations from sdk-test-data on different URLs.

You can start it with [`npm run start-mock-server`](https://github.com/Eppo-exp/eppo-multiplatform/blob/21526f774f7cd03fdb2d2a29e4a6321a76967eb0/package.json#L12) and pass different base urls to the client:

- `http://127.0.0.1:8378/ufc/api`
- `http://127.0.0.1:8378/obfuscated/api`
- `http://127.0.0.1:8378/bandit/api`